src/server/game/Groups/GroupMgr.cpp

```diff
@@ -91,19 +91,19 @@ void GroupMgr::LoadGroups()
         uint32 oldMSTime = getMSTime();
 
         // Delete all groups whose leader does not exist
-        CharacterDatabase.DirectExecute("DELETE FROM groups WHERE leaderGuid NOT IN (SELECT guid FROM characters)");
+        CharacterDatabase.DirectExecute("DELETE FROM `groups` WHERE leaderGuid NOT IN (SELECT guid FROM characters)");
         // Delete all groups with less than 2 members (or less than 1 for lfg groups)
-        CharacterDatabase.DirectExecute("DELETE groups FROM groups LEFT JOIN ((SELECT guid, count(*) as cnt FROM group_member GROUP BY guid) t) ON groups.guid = t.guid WHERE t.guid IS NULL OR (t.cnt<=1 AND groups.groupType <> 12)");
+        CharacterDatabase.DirectExecute("DELETE FROM `groups` WHERE guid NOT IN (SELECT guid FROM group_member GROUP BY guid HAVING COUNT(guid) > 1)");
         // Delete invalid lfg_data
-        CharacterDatabase.DirectExecute("DELETE lfg_data FROM lfg_data LEFT JOIN groups ON lfg_data.guid = groups.guid WHERE groups.guid IS NULL OR groups.groupType <> 12");
+        CharacterDatabase.DirectExecute("DELETE FROM lfg_data WHERE guid IN (SELECT guid FROM `groups` GROUP BY guid HAVING COUNT(guid) <> 12)");
```

Comment:
same as above

Reply:
I change because otherwise we got worldserver startup errors.
I have not actually tested that change myself

Reply:
Without this change, we get this error in worldserver which prevents it from starting
```
2020-04-16 20:50:54 SQL: DELETE lfg_data FROM lfg_data LEFT JOIN groups ON lfg_data.guid = groups.guid WHERE groups.guid IS NULL OR groups.groupType <> 12
2020-04-16 20:50:54 ERROR: [1064] You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'groups ON lfg_data.guid = groups.guid WHERE groups.guid IS NULL OR groups.groupT' at line 1
```

Reply:
that's why we shouldn't start big PRs like that if not sure

Reply:
@Kitzunu I understand that you changed in order to avoid SQL errors, but I don't understand how the logic is changed here